### PR TITLE
Add the OO_BUNDLE env var to the optional_operators step

### PR DESCRIPTION
**Pull request by dirgim**

### CODE REVIEW

**Comment by petr-muller** [comment]
/hold

I'm not sure if this is that easy, this can interact with ci-op building bundles itself (ci-op does not assume there is a single bundle). I will need to look more closely at this.

File: pkg/steps/optional_operators.go [hunk 1]
```diff
@@ -63,6 +67,9 @@ func resolveOptionalOperator(parameters getter) (*optionalOperator, error) {
 		*valuePointer = value
 	}
 
+	if oo.Bundle, err = parameters.Get(OOBundle); err != nil {
+		oo.Bundle = ""
+	}
```
Comment (petr-muller) [inline pkg/steps/optional_operators.go:72]: Hm, from the changes I saw in the registry I'd say the bundle would be required and not optional, no? It's consumed by the new scorecard step which would be always present, so withtout a bundle the test would always fail....

**Review by petr-muller — APPROVED** [review]
/hold

LGTM and should not interfere with existing OO code. I have a question on the required-ness of the bundle param but if you're sure the param can be optional feel free to merge.

File: pkg/steps/optional_operators.go (hunk 1 above)
Comment (dirgim) [reply pkg/steps/optional_operators.go:72]: That's a good point - adding the change to make it required.

Comment (dirgim) [reply pkg/steps/optional_operators.go]: @petr-muller I made the changes to have the bundle required, ptal

**Review by petr-muller — APPROVED** [review]

**Pull request merged**